src/mumble/TalkingUI.cpp

```diff
@@ -805,6 +809,11 @@ void TalkingUI::on_settingsChanged() {
 
 	const ClientUser *self = ClientUser::get(Global::get().uiSession);
 
+	// Update Background Color
+	if (Global::get().s.qcTalkingUI_BackgroundColor.isValid()) {
+		setStyleSheet(QString("background-color: %1;").arg(Global::get().s.qcTalkingUI_BackgroundColor.name()));
+	}
+
```

Comment:
I don't really like that we have this code twice in TalkingUI.cpp. Maybe we should create a new method and call that in both places :thinking: